lighthouse-core/lib/stack-packs.js

```diff
@@ -7,6 +7,10 @@
 
 const stackPacks = require('@lighthouse/stack-packs');
 
+const requiredStacks = new Map([
+  ['wordpress', ['js:wordpress']]
```

Comment:
I kinda prefer the format @paulirish outlined since it's more explicit, WDYT?

Reply:
updated it to make it more verbose